api/client/client.go

```diff
@@ -215,16 +215,32 @@ func collectionConfigToPb(c db.CollectionConfig) (*pb.CollectionConfig, error) {
 	}, nil
 }
 
-// GetDBInfo retrives db addresses and keys.
-func (c *Client) GetDBInfo(ctx context.Context, dbID thread.ID, opts ...db.ManagedDBOption) (*pb.GetDBInfoReply, error) {
+// GetInviteInfo retrives db addresses and keys.
+func (c *Client) GetInviteInfo(ctx context.Context, dbID thread.ID, opts ...db.ManagedDBOption) ([]ma.Multiaddr, thread.Key, error) {
```

Comment:
Could this be `GetDBInvite`, so that it's clear it returns a DB invite, as apposed to a collection invite or something else?

Also, you might use this for getting the key.. as in, not when you need an invite. That's why I had it more generic: `GetDBInfo`

Reply:
Ah didn't realize there were other use cases. In that case, I'll just change it back to GetDBInfo.